lib/WebService/Async/Onfido/Base/Address.pm

```diff
@@ -50,23 +50,23 @@ sub building_name : method { shift->{ building_name } }
 
 =head2 street
 
-The street of the applicant’s address. There is a 32-character limit on this field for UK addresses.
+The street of the applicant's address. There is a 32-character limit on this field for UK addresses.
```

Comment:
These are autogenerated from the API documentation - you'd need to update the script if you want these changes to be permanent.

Reply:
Where is the script ?

Reply:
I added `=encoding utf8` at the beginning part in that script.